scaleway/resource_rdb_instance_beta.go

```diff
@@ -140,9 +185,29 @@ func resourceScalewayRdbInstanceBetaRead(d *schema.ResourceData, m interface{})
 	d.Set("user_name", d.Get("user_name").(string)) // user name and
 	d.Set("password", d.Get("password").(string))   // password are immutable
 	d.Set("tags", res.Tags)
+	if res.Endpoint != nil {
```

Comment:
If res.Endpoint is null you should set the value to empty string.

Reply:
Should I set `endpoint_port` value to zero as well? 🤔

Reply:
yes